docs/processes/release/tasks/versions.md

```diff
@@ -92,6 +92,126 @@ Edit icuver.txt directly.
+        UNICODE_7_0 = getInstance(7, 0, 0, 0);
+    <b>
+        ICU_VERSION = getInstance(54, 1, 0, 0);
+        ICU_DATA_VERSION = ICU_VERSION;</b>
+        UNICODE_VERSION = UNICODE_7_0;
+
+        UCOL_RUNTIME_VERSION = getInstance(8);
+        UCOL_BUILDER_VERSION = getInstance(9);
+        UCOL_TAILORINGS_VERSION = getInstance(1);
+    }
+    </pre>
+
+    In the same file, starting at line 164 (as of 54.1) -
+
+    <pre>
+    /**
+     * Data version string for ICU's internal data.
+     * Used for appending to data path (e.g. icudt43b)
+     * @internal
+     * @deprecated This API is ICU internal only.
+     */
+    @Deprecated
+    public static final String <b>ICU_DATA_VERSION_PATH = "54b";</b>
+    </pre>
+
+2. When creating the final release of a major ICU version, 
+or a point release (minor version update on a maintenance branch),
+update the Maven project version for ICU4J at the root and all of the submodules
+with the follwing Maven command.
+The command requires a version number string that follows the typical Java / Maven version number conventions.
+
+    The following examples illustrate the version numbers to use during and after the ICU 74 release:
+
+    * For an ICU major version change's final release, use: `74.1.0`
+    * On the `main` branch in the commit after the commit from which the release branch was cut,
+      use: `75.0.1-SNAPSHOT`
+    * For a point release (ICU minor version change), use: `74.2.0`
+    * On the maintenance branch (ex: `maint/maint-74`) in the commit after the commit creating the point release, use: `74.2.1-SNAPSHOT`
+
+    **Note:** during RC releases, we update the version to be the final release version.
+    We do so to frontload work that would be otherwise done during the final/GA release.
+
+    With the proper new version number, run the following Maven command to update the version numbers
+    throughout all of the ICU4J `pom.xml`` build files.
+
+    ```
+    mvn versions:set -DnewVersion=75.1.0 -DgenerateBackupPoms=false
+    ```
+
+3. Immediately after creating the final release of a major ICU version, update the value that represents just the major version number of the semantic version. To do this, update the value of the ICU `icu.major.version` property in the root Maven pom.xml file at `icu4j/pom.xml`.
+
+    This can be done by running the following command:
+
+    ```
+    mvn versions:set-property -DnewVersion=75 -Dproperty=icu.major.version
+    ```
+
+    This should happen at the same time and along with the work in the previous step for the version number
+    when the version number is updated on `main` in the commit after the release/maintanence branch is cut.
+    In other words, the above `versions:set-property` step should be executed at the same time
+    `mvn versions:set -DnewVersion=75.0.1-SNAPSHOT` is executed.
+
+4. Update the `api.doc.version` property.
+Edit the root pom file at `icu4j/pom.xml` and manually make this change:
```

Comment:
TBD: we might consider using the version directly instead of `${icu.major.version}`.

That way one can change it from command line:
```
mvn versions:set-property -DnewVersion="75 Release Candidate" -Dproperty=api.doc.version
```

Or maybe remove "Release Candidate" completely.
"In my book" RC means: if there are no problems found then this is promoted to GA.

Using "Release Candidate" anywhere means that even if everything is perfect, one has to remove such  and rebuild in order to release.

Reply:
Note: not actionable for this PR, of course.